DemoApp/controller/user.controller.js

```diff
@@ -60,33 +58,38 @@ const userRegister=async(req,res,employeetype)=>{
     });
     console.log("Saved")  
     await NewUser.save();
-    console.log("Saved")
+    const file_upload = await new Report({
+      // filepath: filepath,
+      u_id: NewUser._id,
+    }).save();
     // return successResponse(req,res,NewUser,SUCCESSFULLY_REGISTERED,200)
     return res.redirect('/login');
         
     } catch (error) {
       console.log(error)
         return errorResponse(req,res,SOMETHING_WENT_WRONG,500)
     }
-
+  
 }
 
 // for login
-const userLogin=async(req,res,employeetype)=>{
+const userLogin=async(req,res)=>{
     let userCred = req.body
-    console.log(userCred)
+    
     let {username,password}=userCred;
-    console.log("dtaaaaa",username)
+    
     const user=await User.findOne({username})
-    console.log("userrr",user)
+    let role = user.role
+
+    console.log("userrr",user.role)
     //check username is in database or not
     if(!user)
     {
         return  errorResponse(req,res,USER_NOT_EXIST,400)
     }
 
     //we will check role
-    if(user.employeetype!=employeetype){
+    if(user.role!=role){
```

Comment:
On line no. 82, you're doing let role = user.role and here on line 92, you're checking if(user.role !=role), this condition will never be false as you're comparing the same values.